src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
Why `splice`? If it's to copy the array, use `slice`. Also, shouldn't processes be removed from the process queue to avoid multiple executions?